Toshi/Source/TCString.cpp

```diff
@@ -100,7 +100,7 @@ bool __thiscall Toshi::TCString::AllocBuffer(int a_iLength, bool freeMemory)
             {
                 tfree(m_pBuffer);
             }
-            m_pBuffer = &m_aNull;
+            m_pBuffer = nullptr;
```

Comment:
Why do this? The symbols of de blob tell &m_aNull

Reply:
for some reason it was defined as static and it was one of the reasons why the project didn't compile, I don't have De Blob installed and at the moment there's no way for me to install it so that is just a quickfix to make the project compileable